### DISCUSSION TITLE
About the bug of tool crontab

### DISCUSSION THREAD

**Discussion by emoooooo** [he3-app/he3-feedback#270] [Bug Report]
### Version
crontab 1.0.5

### Environment
Windows v1.3.23

### Steps to reproduce
![image](https://github.com/he3-app/he3-feedback/assets/31915809/0d62cbb6-848a-4976-8a0b-fe4b1238e3fa)

### What is expected?
周数1 应该表示的是周日

### What is actually happening?
目前周数1显示的是 星期一，另外希望增加展示最近几次的执行时间，如下图：
![image](https://github.com/he3-app/he3-feedback/assets/31915809/3243ae44-a44b-47ad-a54a-76f8fdce3be2)

**Comment by lyzhang1999** [comment]
crontab 0 和 7 都代表周天，1 是周一。

**Comment by lyzhang1999** [comment]
另外关于新功能的反馈已记录：https://github.com/he3-app/he3-feedback/issues/271
